test/framework/awsLambda.test.js

```diff
@@ -35,6 +35,8 @@ let Dashboard = require("../../recipe/dashboard");
 let { createUsers } = require("../utils");
 const { Querier } = require("../../lib/build/querier");
 const { maxVersion } = require("../../lib/build/utils");
+const { AWSRequest } = require("../../lib/build/framework/awsLambda/framework");
+const sinon = require("sinon");
```

Comment:
if this is a new dependency, you need to add it in dev dependcy in package.json

Reply:
No, `sinon` was already added as devDependency.